internal/integration/provision/provision.go

```diff
@@ -142,10 +142,6 @@ func (suite *BaseSuite) TearDownSuite() {
 		// for failed tests, produce crash dump for easier debugging,
 		// as cluster is going to be torn down below
 		suite.provisioner.CrashDump(suite.ctx, suite.Cluster, os.Stderr)
-
```

Comment:
now sure if this needed to be dropped, this just seemed to gather the service logs and print to stdout, support zip  seemed a better option